src/main/java/net/snowflake/ingest/streaming/internal/ParquetFlusher.java

```diff
@@ -162,6 +159,32 @@ private SerializationResult serializeFromJavaObjects(
         parquetWriter.getExtendedMetadataSize());
   }
 
+  private void addFileIdToMetadata(
+      String filePath, long chunkStartOffset, Map<String, String> metadata) {
+    // We insert the filename in the file itself as metadata so that streams can work on replicated
+    // mixed tables. For a more detailed discussion on the topic see SNOW-561447 and
+    // http://go/streams-on-replicated-mixed-tables
```

Comment:
nit: pl add the other golink here too from line 133 in old code

Reply:
Done.